imagersite/imagerprofile/models.py

```diff
@@ -17,6 +18,14 @@ class ImagerProfile(models.Model):
         related_name="profile",
         null=False,
     )
+    photos = models.ManyToManyField(
+        Photos,
+        related_name='user_photos',
+    )
+    albums = models.ManyToManyField(
+        Album,
+        related_name='user_albums',
+    )
```

Comment:
It's an interesting choice to make the profile related to the Photo and Album, rather than placing these fields on the Photo and Album.  Implicitly it creates a dependency on the photo app from this profile app.  Is that what you want?  You can see how the initial migration for this app now has a dependency on the initial migration from the Photo app.